### ISSUE TITLE
Is the domain down?

### ISSUE THREAD

**Issue by umarcor** [fossi-foundation/wishbone#23]
[wishbone-interconnect.org](https://www.wishbone-interconnect.org) seems to de down. Moreover, `fossi-foundation.github.io/wishbone` seems to be redirected. Fortunately, the site is available at [wishbone-interconnect.rtfd.io](https://wishbone-interconnect.rtfd.io).

**Comment by zuberkhan01st** [comment]
Yes its still down

**Comment by JimLewis** [comment]
the spec at wishbone-intecronnect.rftd.io seems to only be B.3.   Any intent on including B.4?

**Comment by rherveille** [comment]
Wishbone releases B.2 and B.3 is are released into the public domain. Currently Fossi hosts the Wishbone B.3 spec.
The Wishbone B.4 spec is a closed source derivative of the B.4 spec. It's authors removed it from the public domain and copyrighted its contents. Therefore the B.4 spec can not be hosted by Fossi and is not endorsed by the Wishbon Spec Steward (me).

**Comment by JimLewis** [comment]
@rherveille   If you are indeed the original and primary author of B.2 and B.3, since B.4 is a derivative of B.3 and since their changes are particularly minimal - the addition of pipelined interface modes, it would be appropriate for you to ask for custody of the license  or to transfer the license to another organization.   

A license that is similar to your original intent - and one that would have enforced your original intent is a creative commons license (https://creativecommons.org/share-your-work/cclicenses/) [CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/).

That said, to have the weight of a standard, there cannot be a multitude of custom derivatives of it.   So the distribution rights that disallow others to edit the document is a good one to have.    It is reasonable that others are encouraged to propose new features to a centralized authority, however, you probably want to have a team that decides if these features are good ones to include or not.   Good ideas might have to be rejected because other good ideas have to take precedence.

**Comment by wallento** [comment]
The domain is back up too

**Comment by JimLewis** [comment]
@wallento  ??    Did you test https://www.wishbone-interconnect.org/ to see if it is up?   I just tested from my location and it says nope.    If your test also fails, please re-open this issue.

**Comment by wallento** [comment]
Ah, I see, the HTTPS version does not work and must be the default. Looking into it!

**Comment by JimLewis** [comment]
@wallento   Have you thought of asking OpenCores if they would move the copyright to FOSSI and then you could address the issues that concern @rherveille?

**Comment by wallento** [comment]
Yes, we had this discussion repeatedly, and unfortunately there is not much success so far. I think in this case its a different (also dissolved) company. It was essentially why we started FOSSi Foundation, tired of small companies wanting to own everything and then just disappear..

**Comment by JimLewis** [comment]
I think copyrights automatically belong to the authors of a document.   If the authors did not sign their copyrights away to OpenCores, then I suspect that they too own rights to the document - and can do what they like with the document - independent of the OpenCores copyright.    

This thought is supported by every US company requiring their employees to sign over their copyrights for everything done while employed by the company to the company.   This prevents authors of work at the company from asserting their copyright over material they do for the company.   

In addition, an author can release the same document under different copyright terms - Gaisler does this with their library and releases their freely downloadable one as GPL and their for sale one under terms that do not require the customer to release any code that is used with the cores.   

Yet I am not a lawyer.
